MonstieWash/Assets/Scripts/Items/ItemLinker.cs

```diff
@@ -0,0 +1,139 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ItemLinker : MonoBehaviour
+{
+    [SerializeField] private LinkMethod link;
+    [SerializeField] private LinkedItem linkObject;
+    private GameSceneManager m_gameSceneManager;
+    private TaskTracker m_taskTracker;
+    private List<ITaskScript> m_taskScript = new();
+    private enum LinkMethod
+    {
+        Group,
+        Single
+    }
+    private void Awake()
```

Comment:
Also very small thing but if you could just add a line of spacing between the field declarations, the enum declaration and the Awake method that would be great to improve readability.

```suggestion
    private List<ITaskScript> m_taskScript = new();
    
    private enum LinkMethod
    {
        Group,
        Single
    }
    
    private void Awake()
```